GETTING_STARTED.md

```diff
@@ -584,6 +584,25 @@ factory :post do
 end
 ```
 
+Attribute overrides can be used to link associated child objects to their parent:
```

Comment:
```suggestion
Attribute overrides can be used to link associated objects:
```

Could we also add a section title here to make it easier to link to this? Something like `### Association Overrides`. We would also need to manually add that section to the index at the top of this file (sorry, I haven't yet found a great way of doing that automatically).